### PR TITLE
i2-high score test

### PR DESCRIPTION
**Pull request by ryandbarnett**
Updates highest total score test to not rely on fixtures or csv.
Adds a check for if the given path exists in each collection from csv method.

### CODE REVIEW

File: lib/games_collection.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ def initialize(path)
   end
 
   def from_csv(path)
+    return if !File.exist?(path)
```
Comment (hashmaster3k) [inline lib/games_collection.rb:15]: Good catch for edge case

File: test/game_statistics_test.rb [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
     }
 
     @stat_tracker = StatTracker.from_csv(locations)
   end
 
   def test_highest_total_score
-    assert_equal 8, @stat_tracker.highest_total_score
+    game1 = MockGame.new(4, 6)
+    game2 = MockGame.new(2, 2)
+    game3 = MockGame.new(0, 3)
```
Comment (hashmaster3k) [inline test/game_statistics_test.rb:22]: More great use of mocks

**Review by hashmaster3k — COMMENTED** [review]
Looks good.

**Review by dhalverson — APPROVED** [review]
We can review the code against the spec harness once we have most of the methods in
